spec/rubocop/cop/style/documentation_method_spec.rb

```diff
@@ -0,0 +1,49 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::DocumentationMethod do
```

Comment:
You need more examples. Methods in modules, methods on the top level (outside a class), different ways of declaring a method private (`private def method` and `private :method`), combinations of public and private methods.